pyproject.toml

```
@@ -69,6 +68,7 @@ shapely = ">=2.0.7"
 spatialist = ">=0.14.0"
 s1etad = ">=0.5.5,<0.6"
```

```suggestion
s1etad = ">=0.5.5"
```

I forgot to remove the upper bound